src/views/account/Login.vue

```diff
@@ -2,28 +2,36 @@
     <div>
         <auth-header></auth-header>
         <div class="container registration">
-            <h1 class="title">Login</h1>
-            <section class="box" lang="en">
-                <b-field>
-                    <b-input placeholder="Email" type="email" v-model="data.email" required></b-input>
-                </b-field>
+            <form name="loginForm" @submit.prevent="login">
```

Comment:
Do we need to handle the submit event if we call the "login" function on button click? And vice versa - do we need to call "login" function on button click if we have the form which handles the submit event?
Let's reduce the handling to one of the methods (either button click or form submission)